services/reproducible-central/reproducible-central.service.js

```diff
@@ -0,0 +1,91 @@
+import Joi from 'joi'
+import { BaseJsonService, pathParams } from '../index.js'
+
+const schema = Joi.object({
+  color: Joi.string().required(),
+  message: Joi.string().required(),
+}).required()
+
+export default class ReproducibleCentral extends BaseJsonService {
+  static category = 'dependencies'
+
+  static route = {
+    base: 'reproducible-central/a',
```

Comment:
What does '/a' mean here?

Reply:
a means "artifactId", to differentiate with eventual future p for "project"

Reply:
I think we should make the URL /artifact then, and the summary "Reproducible Central Artifact"